### PR TITLE
Feature/kc in docker compose

### PR DESCRIPTION
**Pull request by creisle**
- Put keycloak container into docker compose
  - Add default users to real export
  - Download public key with healthcheck
- Add health checks to APIs
- simplify install instructions to remove keycloak manual set up steps
- Add a basic test that the docker-compose will start ok

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -1,122 +1,121 @@
...
+| ipr_graphkb_link | GraphKB       | This is the user used by IPR to pull data from GraphKB                                                                                               |
+| iprdemo          | IPR           | This is an admin user in the IPR demo db                                                                                                             |
+| graphkb_admin    | GraphKB       | Admin user for managing content/users in the GraphKB web interface                                                                                   |
+
+
+![default users](../images/pori-keycloak-default-users.png)
+
+## Run docker-compose
+
+First thing you should do is create new/empty directories for the data stored by GraphKB and IPR.
```
Comment (elewis2) [inline docs/install.md:38]: possibly 'The first thing' instead of 'First thing'

File: docs/install.md [hunk 2]
```diff
@@ -1,122 +1,121 @@
...
+| Name             | Default in DB | Purpose                                                                                                                                              |
+| ---------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| graphkb_importer | GraphKB       | This is the default user that is created when the new GraphKB DB is created. It is an admin user that can be used to add new users or import content |
+| ipr_graphkb_link | GraphKB       | This is the user used by IPR to pull data from GraphKB                                                                                               |
+| iprdemo          | IPR           | This is an admin user in the IPR demo db                                                                                                             |
+| graphkb_admin    | GraphKB       | Admin user for managing content/users in the GraphKB web interface                                                                                   |
+
+
+![default users](../images/pori-keycloak-default-users.png)
```
Comment (mathieulemieux) [inline docs/install.md:34]: images folder inside docs folder. Fix broken link with this: ./images/pori-keycloak-default-users.png

File: docs/developer_reference/getting_started.md [hunk 3]
```diff
@@ -2,56 +2,73 @@
...
 
-Now you are ready to start up with the dev compose yml
+Then you can load these terms using the ontology file loader
 
 ```bash
-docker-compose -f demo/docker-compose.dev.yml up -d
+docker run --net host bcgsc/pori-graphkb-loader:latest \
+    -u graphkb_importer \
+    -p secret \
+    -g http://localhost:8888/api \
```
Comment (elewis2) [inline docs/developer_reference/getting_started.md:70]: should be 8080

File: docs/developer_reference/getting_started.md [hunk 4]
```diff
@@ -2,56 +2,73 @@
...
+    file \
+    ontology \
+    vocab.json
```
Comment (elewis2) [inline docs/developer_reference/getting_started.md:73]: Recap of discussion elsewhere - on testing this out it turned out a --mount arg was needed. Absolute path for vocab.json was needed (when I ran it) for source and dst components of the --mount arg as well as the filename after 'ontology'.

**Review by elewis2 — APPROVED** [review]

**Pull request merged**